azure/build-pipeline.yml

```diff
@@ -50,4 +50,4 @@ jobs:
           jdkArchitectureOption: 'x64'
           publishJUnitResults: true
           testResultsFiles: '**/TEST-*.xml'
-          tasks: 'mingwX64Test'
+          tasks: 'build'
```

Comment:
build will run check which will run the test (as well as tests for any targets that get added in the future)